src/context/chatSelectors.js

```diff
@@ -0,0 +1,13 @@
+import { createSelector } from 'reselect'
```

Comment:
you can use React.memo or useMemo in this case, use don't really use reselect here